js/gdm/loginDialog.js

```diff
@@ -1179,6 +1179,7 @@ const LoginDialog = new Lang.Class({
                              this._promptEntry.reactive = false;
                              return this._beginVerificationForUser(userName);
                          }
+                         return undefined;
```

Comment:
Yes, we really need it, that ensures the function doesn't return until the hold returned by `_beginVerificationForUser` has been canceled. The `ConsecutiveBatch` object ensures this. It's frankly really confusing and most uses of it have since been removed upstream, but it's very hard to get rid of without rebasing on the latest upstream code.